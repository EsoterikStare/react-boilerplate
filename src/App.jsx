import React, { Component } from 'react'
import Checkbox from './Checkbox'

require('./app.css')

export default class App extends Component {
  render () {
    return (
      <div>
        <h1>Hello, world.</h1>
        <div className='box'>Don't put me in a box</div>
        <div>
          <Checkbox labelOn='On' labelOff='Off' />
        </div>
      </div>
    )
  }
}
