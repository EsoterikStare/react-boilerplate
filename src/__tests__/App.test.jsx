/* eslint-env jest, jasmine */
import React from 'react'
import renderer from 'react-test-renderer'
import App from '../App'

test('App displays box', () => {
  const component = renderer.create(
    <App />
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
