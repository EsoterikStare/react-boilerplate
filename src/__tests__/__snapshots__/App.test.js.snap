exports[`test App displays box 1`] = `
<div>
  <h1>
    Hello, world.
  </h1>
  <div
    className="box">
    Don\'t put me in a box
  </div>
</div>
`;
